client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -148,6 +151,9 @@
     @Option(names = "--output-format", paramLabel = "<format>", defaultValue = "CSV", description = "Output format for batch mode [${COMPLETION-CANDIDATES}] " + DEFAULT_VALUE)
     public OutputFormat outputFormat;
 
+    @Option(names = "--pager", paramLabel = "<pager>", description = "Path to the pager program used to display the query results")
```

Comment:
Can we provide default value like in case of historyFile?

Reply:
The default is `less -FXRSn` and because it has arguments, it'll get wrapped in `/bin/sh -c "less -FXRSn"`. That can have some side effects, and I don't want to address this in this PR. Should I extract this commit into a separate PR?

Reply:
Please do.

Reply:
https://github.com/trinodb/trino/pull/16151